prepare/tasks/qa/tasks.py

```diff
@@ -3,8 +3,9 @@
 
 add_to_catalog(
     FormTask(
-        inputs=["context", "context_type", "question"],
-        outputs=["answers"],
+        inputs={"context": "str", "context_type": "str", "question": "str"},
+        outputs={"answers": "List[str]"},
+        prediction_type="Any",
```

Comment:
Should this be "str"?

Reply:
I tried to find `prediction_type` but: `metrics.squad metric does not set the 'prediction_type' parameter so input type checking is not performed`. I can change it to `str` though and see if it passes